oldlog.go

```diff
@@ -97,6 +97,19 @@ func SetLogLevel(name, level string) error {
 	return nil
 }
 
+// GetSubsystems returns a slice containing the
+// names of the current loggers
+func GetSubsystems() []string {
+	subs := make([]string, len(loggers))
+
+	i := 0
+	for k := range loggers {
```

Comment:
A cleaner way to write this would be:
```go
subs := make([]string, 0, len(loggers))

for k := range loggers {
  subs = append(subs, k)
}
```

We allocate an array of length zero, but with an underlying buffer of our expected size, then just use go's builtin append to insert everything